src/Core/Geographic/Extent.js

```diff
@@ -380,24 +379,29 @@ Extent.prototype.set = function set(...values) {
 
 Extent.prototype.union = function union(extent) {
     if (extent.crs() != this.crs()) {
-        throw new Error('unsupported union between 2 diff crs');
+        // If one of them is using Radians
```

Comment:
Bad comment ?

Reply:
No the comment is correct but maybe unclear: we know (because of the above `if` that the 2 crs are different, and if the condition below the comment is true, then the 2 crs are using 4326 => one of them must be in radians.